src/main/java/uk/gov/hmcts/reform/wataskmanagementapi/services/CamundaService.java

```diff
@@ -57,4 +57,23 @@ public void claimTask(String taskId, String subjectId) {
         }
 
     }
+
+    public String unclaimTask(String id) {
+        try {
+            return camundaServiceApi.unclaimTask(id);
+        } catch (FeignException ex) {
+            if (HttpStatus.NOT_FOUND.value() == ex.status()) {
```

Comment:
I think you can catch an exception of type FeignException.NotFound rather than checking this